cppcache/src/PdxInstanceImpl.cpp

```diff
@@ -149,57 +149,50 @@ void PdxInstanceImpl::writeField(PdxWriter& writer,
       break;
     }
     case PdxFieldTypes::BYTE_ARRAY: {
-      if (auto&& val = std::dynamic_pointer_cast<CacheableBytes>(value)) {
-        writer.writeByteArray(fieldName, (int8_t*)val->value(), val->length());
+      if (auto&& val = std::dynamic_pointer_cast<CacheableArray<int8_t>>(value)) {
```

Comment:
Did we lose these typedefs?

Reply:
using CacheableBytes = CacheableArray<int8_t>